spec.html

```diff
@@ -19147,8 +19153,7 @@ <h1>
       <emu-alg>
         1. Let _propertyNameReference_ be ? Evaluation of _expression_.
         1. Let _propertyNameValue_ be ? GetValue(_propertyNameReference_).
-        1. Let _propertyKey_ be ? ToPropertyKey(_propertyNameValue_).
-        1. Return the Reference Record { [[Base]]: _baseValue_, [[ReferencedName]]: _propertyKey_, [[Strict]]: _strict_, [[ThisValue]]: ~empty~ }.
+        1. Return the Reference Record { [[Base]]: _baseValue_, [[ReferencedName]]: _propertyNameValue_, [[Strict]]: _strict_, [[ThisValue]]: ~empty~ }.
```

Comment:
Same change is necessary [for `super`](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-super-keyword-runtime-semantics-evaluation):


```js
class C extends class {} {
  m() {
    super[{toString() { console.log('ToPropertyKey'); } }] = console.log('rhs');
  }
}
(new C).m()
```
prints 'rhs' then 'ToPropertyKey'.